.github/workflows/build_and_test.yml

```diff
@@ -420,19 +420,22 @@ jobs:
     container:
       image: 369495373322.dkr.ecr.eu-central-1.amazonaws.com/build-tools:${{ needs.build-buildtools-image.outputs.build-tools-tag }}
       # Default shared memory is 64mb
-      options: --init --shm-size=512mb
+      options: --init --shm-size=512mb --cgroupns=private --privileged
```

Comment:
TODO security review

Reply:
Asked for review in Slack https://neondb.slack.com/archives/C059ZC138NR/p1706265241134019